### PR TITLE
chore: adds icons to deploy process

### PR DESCRIPTION
**Pull request by redreceipt**
- chore: adds icons to deploy process
- stuff

### CODE REVIEW

File: deploys/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```
Comment (vinnyjth) [inline deploys/.gitignore:4]: 👍  better than .gitkeep

File: scripts/generate-app-graphics.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+const fs = require('fs');
```
Comment (vinnyjth) [inline scripts/generate-app-graphics.js:1]: love it

File: scripts/setup-app.sh [hunk 3]
```diff
@@ -5,11 +5,25 @@ fi
...
+WORDMARK=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.WORDMARK_URL")
+if [ -z "$WORDMARK" ]; then echo "APP.WORDMARK_URL invalid!" && exit 1; fi
 IOS_BUNDLE=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.APPLE_BUNDLE_ID")
+if [ -z "$IOS_BUNDLE" ]; then echo "APP.APPLE_BUNDLE_ID invalid!" && exit 1; fi
 ANDROID_BUNDLE=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.ANDROID_PKG_ID")
+if [ -z "$ANDROID_BUNDLE" ]; then echo "APP.ANDROID_PKG_ID invalid!" && exit 1; fi
 SERVER_URL=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP.SERVER_URL")
+if [ -z "$SERVER_URL" ]; then echo "APP.SERVER_URL invalid!" && exit 1; fi
 GOOGLE_MAPS_KEY=$(curl -s --fail -H "x-api-key: $APOLLOS_API_KEY" -H "accept: text/plain" "$API_URL/config/$CHURCH/APP_GOOGLE_MAPS_API_KEY")
+if [ -z "$GOOGLE_MAPS_KEY" ]; then echo "APP.GOOGLE_MAPS_API_KEY invalid!" && exit 1; fi
```
Comment (vinnyjth) [inline scripts/setup-app.sh:24]: what are the chances we could frontload all these `ifs` at the start, let them all log, and then fail if any of them fires? 

I know its' a bit more work, but it prevents that really annoying "build -> watch log -> fix -> build -> watch log -> fix" cycle if you have a sequential series of variables that need to be set.

Comment (vinnyjth) [reply scripts/setup-app.sh:24]: absolutely love that you are adding these explicit errors

Comment (redreceipt) [reply scripts/setup-app.sh:24]: oh I see, so show all of them at once, good idea

Comment (vinnyjth) [reply scripts/setup-app.sh:24]: yah! so if any of them error, you can see what other ones would error before the script exits. so you have a reference for fixing them all at once

Comment (redreceipt) [reply scripts/setup-app.sh]: fixed

**Review by vinnyjth — APPROVED** [review]
I love it

**Pull request merged**